src/yunohost/tools.py

```diff
@@ -322,6 +322,17 @@ def tools_postinstall(operation_logger, domain, password, ignore_dyndns=False,
     else:
         dyndns = False
 
+    cron_job_file = "/etc/cron.hourly/yunohost-generate-dh-params"
```

Comment:
The max time taken by this needs to be tested and maybe moved to daily, on super small slow board that might be too long.

Reply:
Yes. At the begining I set it to dayly... but didn't know if it was too long or not. :-/

Reply:
As discussed in https://github.com/YunoHost/issues/issues/93 , it's ~1min on a RPi2 so I think hourly is fine. But re-testing it would be good.